### ISSUE TITLE
Quick pointer for Selenium?

### ISSUE THREAD

**Issue by jamesmalin** [Sparticuz/chromium#89]
First, this is a great project and it works excellent with AWS Lambda functions right out of the box. Nice work. 

## What would you like to have implemented?

I was hoping to use this with Selenium instead of Puppeteer.

## Why would it be useful?

It would be useful for those that have large projects already using Selenium. 

-----

Do you know a quick fix to get this working? I saw selenium-webdriver/chromium, but switching to it produces an error. 
Without looking into the nuts and bolts of @sparticuz/chromium, should this be possible with a few adjustments or do you know if there something in the build that would stop this from working? Thanks.

Ex:
require("chromedriver");
const chromium = require("@sparticuz/chromium");
let swd = require("selenium-webdriver");
const fs = require("fs");
let { username, password } = require("./credentials.json");
var chrome = require("selenium-webdriver/chrome");
let browser = new swd.Builder();
var options = new chrome.Options();
options.setBinaryPath("/tmp/chromium");
options.addArguments('--headless');
let tab = browser.forBrowser("chrome").setChromeOptions(options).build();
...

**Comment by Sparticuz** [comment]
I've not used selenium so I'm not sure.

This package's main goal is getting a chromium binary for you to use. So as long as you call `executablePath()` you should get a chromium binary in`/tmp/chromium`

The other functions are just helpers (args).

If I get a change I'll take a look at selenium, if you figure it out, let me know, or add a PR for the examples directory.

**Comment by Sparticuz** [comment]
Try calling `chromium.executablePath()` before the selenium web driver and it may work

**Comment by jamesmalin** [comment]
Hey @Sparticuz, I just got back and tested it out. I see how it's being rolled out and took a look at how this package is extracting chromium. Even after waiting for some reason this isn't playing well with Selenium. I tested by changing the path to my local installation of Chrome as a quick test and that worked. I was able to test yours with Puppeteer in AWS and it worked just fine. Not sure what the problem is atm, but thinking there is a param that needs to be set for Selenium to use chromium.
